tap/extensions/http/structs.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/google/martian/har"
+	"github.com/romana/rlog"
+)
+
+type HTTPPayload struct {
+	Type string
+	Data interface{}
+}
+
+type HTTPPayloader interface {
+	MarshalJSON() ([]byte, error)
+}
+
+type HTTPWrapper struct {
+	Method  string      `json:"method"`
+	Url     string      `json:"url"`
+	Details interface{} `json:"details"`
+}
+
+func (h HTTPPayload) MarshalJSON() ([]byte, error) {
+	switch h.Type {
+	case "http_request":
```

Comment:
maybe those should be consts? (http_request, http_response)

Reply:
@gadotroee turned them into `const`(s) with https://github.com/up9inc/mizu/commit/842122b5a1ce1975af35cb3a01060212c7103447